client/src/components/Workflow/Editor/Lint.vue

```diff
@@ -114,8 +114,8 @@ export default {
         untypedParameters: {
             type: UntypedParameters,
         },
-        nodes: {
-            type: Object,
+        getManager: {
+            type: Function,
```

Comment:
Is this better than passing in the nodes ? Can it ever be reactive ?

Reply:
In my experience this is much safer then parsing an object containing all node Vue components and declaring the entire object as reactive. This gives us more control. The function itself does not change but the computed values which access specific attributes are reactive. Nevertheless this is a transitional feature, once the nodes are pure objects, we can switch it over and declare the object as reactive.